L1Trigger/TrackFindingTracklet/interface/PurgeDuplicate.h

```diff
@@ -40,19 +40,15 @@ namespace trklet {
     bool isSeedingStub(int, int, int);
     std::string l1tinfo(const L1TStub*, std::string);
     std::pair<int, int> findLayerDisk(const Stub*);
-    std::vector<double> getInventedCoords(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for prompt tracking
-    std::vector<double> getInventedCoordsExtended(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for extended tracking
-    std::vector<const Stub*> getInventedSeedingStub(
-        unsigned int,
-        Tracklet*,
-        std::vector<const Stub*>);  // return stub with invented x,y,z coords, if it's a seeding one for this tracklet
-
+    // calculate stub coordinates based on tracklet trajectory for prompt tracking
+    std::vector<double> getInventedCoords(unsigned int, const Stub*, Tracklet*);
+    // calculate stub coordinates based on tracklet trajectory for extended tracking
+    std::vector<double> getInventedCoordsExtended(unsigned int, const Stub*, Tracklet*);
+    // return stub with invented x,y,z coords, if it's a seeding one for this tracklet
+    std::vector<const Stub*> getInventedSeedingStub(unsigned int, Tracklet*, std::vector<const Stub*>);
```

Comment:
Function arguments that are pointers should be declared const, unless the function changes them.
e.g. "const Tracklet*" in function "getInventedSeedingStub(...)". Again, you didn't write many of these, but please fix them. @sarafiorendi please make a note of this rule too.

Reply:
This has been done.